samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js

```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
   p,
   makeAdmin,
   removeFromCall,
-  isOwner,
+  localIsOwner,
+  localIsAdmin,
   count,
 }) => (
   <li>
     <span>
       {`${count}. `}
-      <b>
-        {p.permissions.canAdmin && 'Admin | '}
-        {p.local && '(You) '}
-      </b>
+      {p.permissions.canAdmin && <b>{p.owner ? 'Owner | ' : 'Admin | '}</b>}
+      <b>{p.local && '(You) '}</b>
       {p.user_name}: {p.session_id}
     </span>{' '}
-    {!p.local && !p.permissions.canAdmin && isOwner && (
+    {!p.local && !p.owner && (localIsAdmin || localIsOwner) && (
```

Comment:
So the thinking in this code block was... 

1. An owner can remove anyone from the call
1. An admin can remove any non-admin from the call, but NOT an owner
1. An owner or an admin can make anyone who is not an admin an admin. 

WDYT?